openbook_auth/models.py

```diff
@@ -3470,3 +3527,25 @@ def bootstrap_user_auth_token(user):
 
 def bootstrap_user_profile(user, name, is_of_legal_age, avatar=None, ):
     return UserProfile.objects.create(name=name, user=user, avatar=avatar, is_of_legal_age=is_of_legal_age, )
+
+
+class UserNotificationSubscription(models.Model):
```

Comment:
Hmmmmm this should be plural man... We have `UserNotificationsSettings` which also contains multiple boolean settings.

I know it sucks to do these big renames but I'm confident we will not regret doing so later on when more people have to work on this code.

=> `UserNotificationsSubscription`